### PR TITLE
[MySQL] Fix discrepancy between swagger and resource id

### PR DESCRIPTION
**Pull request by DaeunYim**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by DaeunYim** [comment]
Got three errors when autogenerating,
WARNING: Directive with 'where' clause '$' failed to execute transformation 'return $.replace(/\/subscriptions\/\{subscriptionId\}\/resourceGroups\/\{resourceGroupName\}\/providers\/Microsoft\.DBForMySql\/flexibleServers\/\{serverName\}/g, "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DBforMySQL/flexibleServers/{serverName}")' in path ': '$.replace is not a function'
WARNING: Directive with 'where' clause '$' failed to execute transformation 'return $.replace(/\/subscriptions\/\{subscriptionId\}\/resourceGroups\/\{resourceGroupName\}\/providers\/Microsoft\.DBForMySql\/flexibleServers\/\{serverName\}/g, "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DBforMySQL/flexibleServers/{serverName}")' in path ': '$.replace is not a function'
WARNING: Directive with 'where' clause '$' failed to execute transformation 'return $.replace(/\/subscriptions\/\{subscriptionId\}\/resourceGroups\/\{resourceGroupName\}\/providers\/Microsoft\.DBForMySql\/flexibleServers\/\{serverName\}/g, "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DBforMySQL/flexibleServers/{serverName}")' in path ': '$.replace is not a function'

three of them are for (server list by rg, server list by subscription, location capabilities ) which are not directly related to pipelining but let me knowif there's potential issues..

**Review by msJinLei — APPROVED** [review]

File: src/MySql/readme.md [hunk 1]
```diff
@@ -59,6 +59,9 @@ title: MySQL
 subject-prefix: 'MySQL'
 
 directive:
+  - from: swagger-document
```
Comment (msJinLei) [inline src/MySql/readme.md:62]: You are using multiple swagger files. Please specify the swagger file you want to apply the replacement.

**Review by msJinLei — CHANGES_REQUESTED** [review]

File: src/MySql/readme.md (hunk 1 above)
Comment (DaeunYim) [reply src/MySql/readme.md]: Will do

**Review by msJinLei — APPROVED** [review]

**Pull request merged**